docs/application/tizen-studio-0/setup/test.md

```diff
@@ -0,0 +1,135 @@
+# Grid layout(Tizen 6.0 or late)
+
+> **Note**
+>
+> If you are using GridLayout on Tizen 5.5, Please refer to [Grid Layout(Tizen 5.5)](#gridlayout-tizen5.5) instead.
+
+`GridLayout` is a grid box for the two dimentional layout. It constraints the x and y position, width and height of the child actors.
+
+Positioning children in a grid form, the size of each column and row is set to the largest size of cells belonging to their axis.
+
+If the `Column` or the `row` Property is not specified on the child, it is automtically assigned left to right or top to bottom depends on the `GridOrientation` Property.
+
+
+There are two types of properties. One is the properties for the grid layout and Another one is the properties that affect the specified child:
+
+The following table describes the properties of `GridLayout`.
+| Property               | Type            | Description |
+| -----------------------| --------------- | ------------ |
+| `Columns`              | int             | Gets or sets the number of colomns in the grid. |
+| `Rows`                 | int             | Gets or sets the number of rows in the grid. |
+| `GridOrientation`      | Orientation     | Gets or sets the orientation in the grid. |
+| `ColumnSpacing         | `float`         | Gets or sets the distance between columns |
+| `RowSpacing`           | float           | Gets or sets the distance between rows. |
+
+<a name="columns-and-rows"></a>
+## Columns and rows
+
+The `Columns` and the `Rows` properties specify the maximum number of cells on a horizontal or vertical axis.
+
+When the `Column` or the `Row` of child is automatically set without specified value, the value cannot be exceeded the number of main-axis cell which is the `Columns` value on horizontal orientation or the `Rows` value on vertical orientation. But, it can be assigned a value that exceeds the number of cross-axis cell.
```

Comment:
Update sentence as below:

When the `Column` or the `Row` of child is automatically set without specified value, the value cannot exceed the number of main-axis cell which is

Reply:
But doesn't go well in the sentence , so consider changing it. 

However, ....it can be assigned a value that exceeds the number of cross-axis cell.